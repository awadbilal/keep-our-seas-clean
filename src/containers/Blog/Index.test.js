import React from "react";
import renderer from "react-test-renderer";
import Blog from "./Index";

it("renders correctly", () => {
  const tree = renderer.create(<Blog />).toJSON();
  expect(tree).toMatchSnapshot();
});
