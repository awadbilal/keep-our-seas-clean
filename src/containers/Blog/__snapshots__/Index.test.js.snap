// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="main pt-5 pb-5 fdb-block container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8 offset-md-2 text-center col"
      >
        <h1
          className="title"
        >
          blogHeader
        </h1>
      </div>
    </div>
  </div>,
  <div
    className="pt-5 pb-3 container"
  >
    <div
      className="row"
    />
    <div
      className="justify-content-center row"
    />
  </div>,
]
`;
